### ISSUE TITLE
Conflict with Prettier

### ISSUE THREAD

**Issue by EvgenyOrekhov** [NullVoxPopuli/eslint-plugin-decorator-position#289]
Prettier (with default options) wraps long decorators to the next line:

```js
export default class Foo {
  @bar("baaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaz")
  qux;
}
```

But then eslint-plugin-decorator-position (with default options) starts saying `Expected @bar to be inline`.

I would prefer to not have this warning when using default options.

**Comment by NullVoxPopuli** [comment]
Thanks for reporting! <3 Any chance you want to submit a failing test PR?

**Comment by NullVoxPopuli** [comment]
This has been resolved!
